### ISSUE TITLE
v5.5.5 Incompatibility with Expokit

### ISSUE THREAD

**Issue by zanechua** [invertase/react-native-firebase#2363]
## Issue

v5.5.5 that was just released is incompatible with the latest expokit. Throws an error about customRewardString in AdMobRewardedVideo not being found due to the library for AdMob being a lower version (Google-Mobile-Ads-SDK v7.22.0). I don't think this is really a bug on react-native-firebase side since all it should need is a library update but should probably mention that v5.5.4 is the last working version for Expokit v33.

---

### iOS

<details><summary>Click To Expand</summary>
<p>

#### `ios/Podfile`:

- [ ] I'm not using Pods
- [x] I'm using Pods and my Podfile looks like:

```ruby
# N/A
```

#### `AppDelegate.m`:

```objc
// N/A
```

</p>
</details>

---

### Android

<details><summary>Click To Expand</summary>
<p>

#### Have you converted to AndroidX?

- [ ] my application is an AndroidX application?
- [ ] I am using `android/gradle.settings` `jetifier=true` for Android compatibility?
- [ ] I am using the NPM package `jetifier` for react-native compatibility?

#### `android/build.gradle`:

```groovy
// N/A
```

#### `android/app/build.gradle`:

```groovy
// N/A
```

#### `android/settings.gradle`:

```groovy
// N/A
```

#### `MainApplication.java`:

```java
// N/A
```

#### `AndroidManifest.xml`:

```xml
<!-- N/A -->
```

</p>
</details>

---

## Environment

<details><summary>Click To Expand</summary>
<p>

**`react-native info` output:**

```
 OUTPUT GOES HERE
```

- **Platform that you're experiencing the issue on**:
  - [ ] iOS
  - [ ] Android
  - [ ] **iOS** but have not tested behavior on Android
  - [ ] **Android** but have not tested behavior on iOS
  - [ ] Both
- **`react-native-firebase` version you're using that has this issue:**
  - `e.g. 5.4.3`
- **`Firebase` module(s) you're using that has the issue:**
  - `e.g. Instance ID`
- **Are you using `TypeScript`?**
  - `Y/N`
  
</p>
</details>

---

Think `react-native-firebase` is great? Please consider supporting all of the project maintainers and contributors by donating via our [Open Collective](https://opencollective.com/react-native-firebase/donate) where all contributors can submit expenses. [[Learn More]](https://invertase.io/oss/react-native-firebase/contributing/donations-expenses)

- 👉 Check out [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter for updates on the library.

**Comment by GioLogist** [comment]
+1 on this issue

**Comment by KowalskiP** [comment]
+1

**Comment by mikehardy** [comment]
@GioLogist '+1's in the open source world are just comment spam as they generate email without doing anything useful. Either add a reaction or do the investigation to find the incompatibility and propose a PR please

**Comment by zanechua** [comment]
@mikehardy

This actually should be more like a notice listed in the README. Don't think there's any PR that's possible since it's a breaking change. The PR would be to revert to 5.5.4.

**Comment by mikehardy** [comment]
@zanechua we have formally provided notice that breaking changes in the *minor* versions are possible with v5.x.x of react-native-firebase, such that the codeline may stay revelant during the v5->v6 transition. So even if it is a breaking change, it is possible to provide a PR, it just needs to have really good upgrade / changelog information. This is in the RNFB docs

**Comment by GioLogist** [comment]
@mikehardy The purpose of the +1 was to both subscribe myself and resurrect the thread - which seems to have worked. Alternative in the “open source world” is getting an automated “no activity in this issue, closing”. Thanks for the lesson, though 🤘🏻

**Comment by mikehardy** [comment]
Subscribe is a button on the right, resurrecting on the thread is putting in the personal effort to contribute - we're no closer to a solution

**Comment by zanechua** [comment]
This issue has been resolved as of expo/expo@6f64291
